doc/guide/julia_quickstart.hpp

```diff
@@ -0,0 +1,192 @@
+@code{.julia}
+using CSV
+using DataFrames
+using Libz
+using mlpack
+
+# Load the dataset from an online URL.  Replace with 'covertype.csv.gz' if you
+# want to use on the full dataset.
+df = CSV.read(ZlibInflateInputStream(open(download(
+        "http://www.mlpack.org/datasets/covertype-small.csv.gz"))))
+
+# Split the labels.
+labels = df[!, :label][:]
+dataset = select!(df, Not(:label))
+
+# Split the dataset using mlpack.  The output comes back as a dictionary,
+# which we'll unpack for clarity of code.
+test, test_labels, train, train_labels = mlpack.preprocess_split(
+    input=dataset,
+    input_labels=labels,
+    test_ratio=0.3)
+
+# Train a random forest.
+rf_model, _, _ = mlpack.random_forest(training=train,
+                              labels=train_labels,
+                              print_training_accuracy=true,
+                              num_trees=10,
+                              minimum_leaf_size=3)
+
+# Predict the labels of the test points.
+_, predictions, _ = mlpack.random_forest(input_model=rf_model,
+                                         test=test)
+
+# Now print the accuracy.  The third return value ('probabilities'), which we
+# ignored here, could also be used to generate an ROC curve.
+correct = sum(predictions .== test_labels)
+print("$(correct) out of $(length(test_labels)) test points correct " *
+    "($(correct / length(test_labels) * 100.0)%).\n")
+@endcode
+
+We can see that we achieve reasonably good accuracy on the test dataset (80%+);
+if we use the full @c covertype.csv.gz, the accuracy should increase
+significantly (but training will take longer).
+
+It's easy to modify the code above to do more complex things, or to use
+different mlpack learners, or to interface with other machine learning toolkits.
+
+@section julia_quickstart_whatelse What else does mlpack implement?
+
+The example above has only shown a little bit of the functionality of mlpack.
+Lots of other commands are available with different functionality.  A full list
+of each of these commands and full documentation can be found on the following
+page:
+
+ - <a href="https://www.mlpack.org/doc/mlpack-git/julia_documentation.html">Julia documentation</a>
+
+You can also use the Julia REPL to explore the @c mlpack module and its
+functions; every function comes with comprehensive documentation.
+
+For more information on what mlpack does, see https://www.mlpack.org/.
+Next, let's go through another example for providing movie recommendations with
+mlpack.
+
+@section julia_quickstart_movierecs Using mlpack for movie recommendations
+
+In this example, we'll train a collaborative filtering model using mlpack's
+<tt><a href="https://www.mlpack.org/doc/mlpack-git/julia_documentation.html#cf">cf()</a></tt> method.  We'll train this on the MovieLens dataset from
+https://grouplens.org/datasets/movielens/, and then we'll use the model that we
+train to give recommendations.
+
+You can copy-paste this code directly into Python to run it.
```

Comment:
Hi, @rcurtin I think so it must be `You can copy-paste this code directly into Julia to run it.`

Reply:
Nice catch, thanks.  Fixed in be48606c.